apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V105__rda_0_12_0_hotfix2.sql

```diff
@@ -0,0 +1,3 @@
+-- Hot fix for the 0.12 RDA API update, remove not null constraint on clm_typ_ind
+
+alter table rda.fiss_claims alter column clm_typ_ind drop not null;
```

Comment:
Did we want to truncate `rda.message_errors` when we're in a fixing context like this, too?

Reply:
The idea is that the DLQ logic should kick in and check the claims that errored, and they should now process, which should set them as `RESOLVED`, then the normal ingestion process would resume.  The errors would then continue to live in the database in the `RESOLVED` state, not affecting any other processes.

We don't currently have a process to clean up our `rda.message_errors` table, it could be through a migration, but that could lead to some clutter in the migrations over time, so it may be worth a discussion.

Reply:
I asked that the table be truncated in TEST environment last Friday while trying to troubleshoot the start up issues we ran into.  In the end it turned out to be the `oneof` related issues that was preventing startup rather than the records in the `message_errors` table.

Reply:
Thanks for weighing-in here folks! I thought that could be a potential _learning_ from the activities on Friday... maybe it was just thrashing in the end.